src/ahbicht/content_evaluation/evaluators.py

```diff
@@ -36,7 +37,18 @@ def __init__(self):
         initializes a cache with all evaluation methods defined in the (child) class
         """
         self._evaluation_methods: Dict[str, Callable] = {}
-        for candidate in inspect.getmembers(self, inspect.ismethod):
+        try:
+            candidates = inspect.getmembers(self, inspect.ismethod)
+        except inject.InjectorException as injector_exception:
+            if str(injector_exception) == "No injector is configured":
+                # this is due to https://github.com/ivankorobkov/python-inject/issues/77
```

Comment:
In dem issue wird nur python 3.8 erwähnt. Aber es ist ein Problem seit 3.8 und besteht weiterhin oder?

Reply:
ja.